bin/www

```diff
@@ -88,3 +88,10 @@ function onListening() {
     : 'port ' + addr.port;
   debug('Listening on ' + bind);
 }
+
+const mongoose = require('mongoose')
+mongoose.Promise = global.Promise
+const connection = 'mongodb://localhost/lizardboard'
+
+mongoose.connect(connection)
```

Comment:
This needs to be moved such that it happens prior to server startup.  Please move imports to the top of the module, with other imports.  We will also need the connection string to be added in a .env file, with instructions added to README on how to set this up.
```js
server.on('error', onError);
server.on('listening', onListening);

mongoose.connect( process.env. MONGODB_URI )
  .then( () => server.listen( port ) )
server.listen(port);
```